config_defaults.yml

```diff
@@ -524,6 +524,8 @@ Gemfile:
           - mswin
           - mingw
           - x64_mingw
+      - gem: puppet-blacksmith
+        version: '~> 3.4'
```

Comment:
This has to have a guard that will only install on ruby platforms or it will break windows.

Reply:
Fixed :)